portfolio/pom.xml

```diff
@@ -23,6 +23,11 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+  		<groupId>com.google.code.gson</groupId>
```

Comment:
Indentation?

Reply:
![image](https://user-images.githubusercontent.com/48307028/83462179-271fc380-a430-11ea-9360-7f842367878c.png)
 oh whoa I'm not sure why that shows up differently

Reply:
Likely because of spaces vs tab characters :)

Reply:
ahh I think I set up tab to be 2 spaces instead of 4 but I tried fixing it (I checked the file and the indent is less) but please let me know if it still looks bad! thank you so much!